.github/workflows/ruby.yml

```diff
@@ -46,11 +46,10 @@ jobs:
           if [ "$DB" = "mysql" ]; then sudo systemctl start mysql.service; fi
           if [ "$DB" = "mysql" ]; then mysql -e 'create database IF NOT EXISTS fast_inserter;' -uroot -proot; fi
       - name: Set up Ruby
-        # Pin to this commit: v1.82.0
```

Comment:
@roland-at-handshake, any reason to pin to this version? the workflow wasn't able to run successfully but using the latest fixes it.

Reply:
I pinned it to give more control over the action that was used. For example, we wouldn't auto-upgrade to a version that introduced a new vulnerability, or was somehow backwards incompatible (thereby breaking us without any changes on our part). I don't feel too strongly about it, but that was my rationale.